### ISSUE TITLE
Separate out darwin core into imports

### ISSUE THREAD

**Issue by cmungall** [BiodiversityOntologies/bco#95]
If I want to import bco, is the intent that I use dwc classes as if they were other obo classes?

Or is the intent to provide a mapping, similar to the equivalence axioms distributed alongside mondo, uberon to non obo-resources?

Would it make sense to separate dwc into a separate import, so that people using bco wouldn't accidentally end up using dwc classes?

**Comment by ramonawalls** [comment]
Good point, @cmungall. We do in fact import a separate file for Darwin Core classes, but merge them during the release. That same file includes all of the Darwin Core properties, which we use all the time with BCO applications, so I want to keep those. 

Do you suggest that we keep the import file as part of the BCO release process but don't merge it, or maintain a separate file in our repo that people can import if they want? That separate file would include the mappings (equivalency axioms) then, instead of the main BCO file. 

Can you please point me to one of the mapping files distributed e.g., with UBERON so I can see how you do it?

**Comment by cmungall** [comment]
I'm not familiar enough with all use cases to definitively advise, but one thing that would be useful would be to include a bco-base.owl module that is just bco entities and axioms about those entities

You can see the bridge ontologies for uberon here: http://uberon.github.io/downloads.html#bridge

**Comment by ramonawalls** [comment]
@cmungall My intent is to make separate OWL files for DwC, one with all DwC classes and dwc:terms (as data properties) and another with all dwciri terms (which are object properties), plus a bridging files to specify the axioms relating dwc terms to BCO terms (e.g., equivalency between BCO classes and dwc classes). The files could then live in the `imports` directory, with IRIs of the form http://purl.obolibrary.org/obo/bco/imports/dwcterms.owl, etc. I could also make a `modules` directory and store them there, but these are not really modules of BCO, and they don't follow ODPs. Does it make more sense to have the bridging files in the import directory, or to create a `bridge_file` directory? There does not seem to be a standard way that I can find.

Since the term sources are not in OBO library, I am not sure if I can use the make file to build the imports, but I will at least try to script it. Is there an option to set use the makefile to import from some other endpoint? I'm fairly confident that I can figure out how to get the make file to import these ontologies once they are made, and to create versions of BCO with and without them.

Does this plan sound reasonable? Any advice on improvements?

**Comment by ramonawalls** [comment]
Part of this process needs to be deciding whether to import DwC object properties. We are currently only importing data properties.
